wiki/BBCode/de.md

```diff
@@ -0,0 +1,304 @@
+# BBCode
+
+**BBCode** ist eine [Auszeichnungssprache](https://de.wikipedia.org/wiki/Auszeichnungssprache) (Markup), die im osu!-Forum und, in einem größeren Umfang, in den meisten Foren im Internet genutzt wird. Damit die Rich-Text-Formatierung möglich wird, umranden Tags Text, um die Formatierung, Attribute, Einbettung usw. festzulegen. Es wird auf verschiedenen Seiten auf der osu!-Webseite verwendet, wie z. B. Forum-Beiträge, Signaturen, Benutzerseiten und Beatmapbeschreibungen.
+
+![Der Forum-Beitragseditor mit seinen Buttons](img/editor-DE.jpg?1 "Die Box zum Editieren in den Foren")
+
+## Verhalten
+
+Wenn vor dem Drücken des Markup-Buttons kein Text markiert wurde, wird um den Cursor ein öffnendes sowie schließendes Tag im Beitragseditor hinzugefügt. Sofern Text vorher markiert wurde, werden die Tags um den markierten Text gesetzt.
+
+Benutzer, die die Formatierung in einem einzigen Textabschnitt setzen möchten, können BBCode-Tags um die Sektion setzen. Allerdings muss die Reihenfolge und die Verschachtelung der Tags **beachtet werden**. Ansonsten wird der Text nicht formatiert.
+
+Beispiele für korrekte und inkorrekte Benutzungen von verschachtelten Tags ist unten näher beschrieben:
```

Comment:
```suggestion
Beispiele für die korrekte und inkorrekte Benutzung von verschachtelten Tags sind unten näher beschrieben:
```

* `Beispiele` is plural, therefore use `sind` instead of `ist`
* I think having 'Benutzung' is a little more elegant. Feel free to take it or leave it.